### ISSUE TITLE
Any plans to suport a timezone setting?

### ISSUE THREAD

**Issue by fiote** [wakatime/sublime-wakatime#86]
Request:
Add a timezone setting that accept timezone-strings as value (ex: America/Edmonton, Europe/Dublin, etc). So instead of uploading the time worked on my timezone, it would upload it on the specified one. This can be achieved by changing the timezone of my computer, but I kinda need it to be project-specific.

Reasoning:
Most of my work is done on 'local time', but some of it it's for a client in another timezone, since I slice my payments on days worked, sometimes my work 'session' goes through my midnight and messes with the payment calculation.

**Comment by alanhamlett** [comment]
Hmm, timestamps are sent up as UTC so this setting would need to be a timezone per project on your dashboard.

**Comment by fiote** [comment]
@alanhamlett Yes, the idea is to have it per project. It could be done on the dashboard side (to group, filter and format the data as another timezone) or on the plugin side (so it would send UTC dates 'in the past' or 'in the future').
